aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -418,6 +418,70 @@ func TestAccAWSElasticSearchDomain_internetToVpcEndpoint(t *testing.T) {
 	})
 }
 
+func TestAccAWSElasticSearchDomain_AdvancedSecurityOptions_UserDB(t *testing.T) {
+	var domain elasticsearch.ElasticsearchDomainStatus
+	ri := acctest.RandInt()
+	resourceId := fmt.Sprintf("tf-test-%d", ri)
```

Comment:
replace with `rName := acctest.RandomWithPrefix("tf-acc-test")` and pass it to the config instead for constructing it in both config and test params